src/Security/BeamNG.cpp

```diff
@@ -191,12 +234,76 @@ void LegitimacyCheck() {
             break;
         }
     }
+
+#elif defined(__APPLE__)
+    struct passwd* pw = getpwuid(getuid());
+    std::string homeDir = pw->pw_dir;
+    std::string crossoverBottlesPath = homeDir + "/Library/Application Support/CrossOver/Bottles/";
+    info("Crossover bottles path: " + crossoverBottlesPath);

```

Comment:
Please make this a debug log and check if it exists

Reply:
should be fix and has change..